<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>peliCAN</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_logging">
      <property name="text">
       <string>Logging: Off</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_online">
      <property name="text">
       <string>Connection: Offline</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_signal_def">
      <property name="text">
       <string>Signal definition: None, accepting nothing in raw-view.</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTreeView" name="treeView">
      <property name="font">
       <font>
        <family>Fixed</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>962</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Online"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Data_Graph"/>
    <addaction name="actionShow_Hide_send_signal"/>
   </widget>
   <widget class="QMenu" name="menu_Logging">
    <property name="title">
     <string>&amp;Logging</string>
    </property>
    <addaction name="action_Logging_Active"/>
    <addaction name="action_RAW_csv_to_readable_csv"/>
   </widget>
   <widget class="QMenu" name="menu_Signals">
    <property name="title">
     <string>&amp;Signals</string>
    </property>
    <addaction name="action_Load_signal_definition"/>
    <addaction name="action_Edit_signal_definition"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Help"/>
    <addaction name="separator"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Signals"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Logging"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Online"/>
   <addaction name="separator"/>
   <addaction name="action_Load_signal_definition"/>
   <addaction name="action_Data_Graph"/>
   <addaction name="actionShow_Hide_send_signal"/>
   <addaction name="action_RAW_csv_to_readable_csv"/>
   <addaction name="action_Logging_Active"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Data_Graph">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/chart_line.png</normaloff>:/icons/icons/chart_line.png</iconset>
   </property>
   <property name="text">
    <string>Show/Hide &amp;Data and Graph</string>
   </property>
  </action>
  <action name="action_Logging_Active">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <activeoff>:/icons/icons/bullet_red.png</activeoff>
     <activeon>:/icons/icons/bullet_green.png</activeon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Active</string>
   </property>
  </action>
  <action name="action_Online">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <activeoff>:/icons/icons/lightbulb_off.png</activeoff>
     <activeon>:/icons/icons/lightbulb.png</activeon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Online</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/arrow_right.png</normaloff>:/icons/icons/arrow_right.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Load_signal_definition">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/book_open.png</normaloff>:/icons/icons/book_open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Load signal definition</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/help.png</normaloff>:/icons/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/information.png</normaloff>:/icons/icons/information.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Edit_signal_definition">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Edit signal definition</string>
   </property>
  </action>
  <action name="action_RAW_csv_to_readable_csv">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/page_edit.png</normaloff>:/icons/icons/page_edit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;RAW *.csv to readable *.csv</string>
   </property>
  </action>
  <action name="actionShow_Hide_send_signal">
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/transmit.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Show/Hide send signal</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
